src/classes/Globals.js

```diff
@@ -132,6 +142,10 @@ class Globals {
 			this.BROWSER_INFO.displayName = 'Cliqz';
 			this.BROWSER_INFO.name = 'cliqz';
 			this.BROWSER_INFO.token = 'cl';
+		} else if (this.IS_MOBILE_APP) {
+			this.BROWSER_INFO.displayName = 'Ghostery Mobile';
```

Comment:
hey @valmik-ghostery ! Would this work for you?

Reply:
Yeah works for me. Just a couple of things.

1 - Not sure about this but you might need to add the token here as well since that's what pulled into the telemetry database.

2 - We need to differentiate between iOS and android if possible. Will this allow us to do that?

Reply:
1- what token do you mean?
2- we do not run this extension on iOS at all. We might need to add a different way to measure stuff there. Should we implement something from scratch?

Reply:
1- I added something random there, we can change it to whatever makes more sense for you :) if ```ghosty``` and/or ```gh``` is not good we can definitely change 

2- great! yes, we should have anolysis there so all good ;)

Reply:
Awesome. Thanks!

1 - can you use 'an' for android. That will fit in with what we used earlier. But anything is fine as long as it's not 'ff'.

Reply:
@valmikkpatel definitely! just updated the PR to have the token ```an```

Are the other values fine? ```ghosty``` and ```Ghostery Mobile``` ?

Reply:
Yes the other values are fine. Thanks!